### DISCUSSION TITLE
Reboot link

### DISCUSSION THREAD

**Discussion by mrsparkle331** [outdoorbits/little-backup-box#177] [Ideas]
When you use the reboot link in the webui (http://X.X.X.X/?reboot=true) the reboot is successful but the next time the pi comes back up you may get another reboot if the user does not close out the browser window or retype the IP without the reboot=true part. Some browsers will try to reload the URL when it sees the IP back up. or a user might try to use the same browser window to access the webui.

I was chasing my tail for a bit because I was seeing the ping responses from the pi but then they would stop. It turned out the browser was reloading the reboot URL in the background.

Is there another method that could be used to reboot via webui that doesnt leave a pending reboot command in the URL bar?

**Comment by outdoorbits** [comment]
The reboot via menu must be link-based, so if your browser retries, it 
would repeat the action. At the bottom of most pages there are buttons 
for shutdown und reboot. These are form-based and would not repeat the 
action.

**Reply by mrsparkle331** [reply]
excellent, I did not know about the button options.
